libs/model-assessment/src/lib/ModelAssessmentDashboard/ModelAssessmentDashboardProps.ts

```diff
@@ -133,6 +133,10 @@ export interface IModelAssessmentDashboardProps
     request: any[],
     abortSignal: AbortSignal
   ) => Promise<any[]>;
+  requestSplinePlotDistribution?: (
+    request: any,
+    abortSignal: AbortSignal
+  ) => Promise<any>;
```

Comment:
Can we update this any with interface of spline data from api?

Reply:
Somehow adding the type here is causing the type to fail. Will leave this as any for now. Looks like most APIs have any as the return type anyways.

Reply:
I am still not okay with any here if it can be removed with interface similar to backend and solving following errors.